src/main/java/picard/vcf/SplitVcfs.java

```diff
@@ -60,7 +65,7 @@
     public File INDEL_OUTPUT;
 
     @Argument(shortName = "D", doc = "The index sequence dictionary to use instead of the sequence dictionaries in the input files", optional = true)
-    public File SEQUENCE_DICTIONARY;
+    public String SEQUENCE_DICTIONARY;
```

Comment:
Is there a reason not to use `PicardHtsPath` here ? Then I think you wouldn't need to add the try/catch block where it's used below to get the sequence dictionary.

Reply:
Good catch